src/utils/helpers.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+interface ILinkProps {
+  href: string;
+  content: string;
+  classes: string;
+}
+
+export function CustomLink(props: ILinkProps) {
+  const history = useHistory();
+
+  return <a href={props.href} className={props.classes} onClick={e => {
+    e.preventDefault();
+    history.push(props.href);
```

Comment:
```suggestion
    history.push(props.href);
    return false;
```

just for older or odd browsers